File: ASCII.cpp
#include <iostream>

using namespace std;

int main() {
	char a = 0;

	system("clear");
	cout << "Number | Hex | Char" << endl;
	for (unsigned short i = 1; i < 260; i++) {
		a = i;
		cout << dec << i << " | 0x" << hex << i << " | " << a << endl;
	}
	return 0;
}

File: ASCII.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import os
i = 0
os.system("clear")
print("Number | Hex | Char")
while (i < 260):
    print("%03d | " % i + "0x%03x" % i + " | " + "%c" % i)
    i += 1
